lib/msf/ui/debug.rb

```diff
@@ -0,0 +1,258 @@
+# -*- coding: binary -*-
+# frozen_string_literal: true
+
+module Msf
+  module Ui
+    ###
+    #
+    # Displays Metasploit information useful for Debugging.
+    #
+    ###
+    module Debug
+      COMMAND_HISTORY_TOTAL = 50
+      ERROR_TOTAL = 10
+      LOG_LINE_TOTAL = 50
+      ISSUE_LINK = 'https://github.com/rapid7/metasploit-framework/issues/new'
+      PREAMBLE = <<~PREMABLE
+          Please provide the below information in any Github issues you open. New issues can be opened here #{ISSUE_LINK.dup}
+          %red%undENSURE YOU HAVE REMOVED ANY SENSITIVE INFORMATION BEFORE SUBMITTING!%clr
+
+          ===8<=== CUT AND PASTE EVERYTHING BELOW THIS LINE ===8<===
+
+
+      PREMABLE
+
+
+
```

Comment:
Looks like there's some extra whitespace here that can be cleaned up into a single line :+1: